File: go/pengyou/model/entity/tag.go
package entity

type Tag struct {
	ID          uint   `gorm:"primaryKey;autoIncrement" json:"id"`
	Name        string `gorm:"size:63" json:"name"`
	Description string `gorm:"size:255" json:"description"`
}

func (table *Tag) TableName() string {
	return "tag"
}

File: java/main/common/src/main/java/com/pengyou/model/entity/Tag.java
package com.pengyou.model.entity;

import org.babyfish.jimmer.sql.*;


import org.jetbrains.annotations.Nullable;

import java.util.List;

/**
 * Entity for table "tag"
 */
@Entity
public interface Tag {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY
    )
    long id();

    @Key
    String name();

    @Nullable
    String description();

    @ManyToMany(mappedBy = "tags")
    List<User> user();

}

